lib/PuppeteerSharp.Tests/ElementHandleTests/ClickTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.ElementHandleTests
```

Comment:
@Meir017 Let's start moving to this naming convention for folders/namespace so we're not forced to use the `PuppeteerSharp.` full-qualified name to use Puppeteer classes.